Gruntfile.js

```diff
@@ -101,7 +94,7 @@ module.exports = function ( grunt ) {
 	grunt.registerTask(
 		'build',
 		'Tests files, moves them to the /build directory, and minifies CSS.',
-		[ 'clean:build', 'test', 'lint', 'copy', 'concat:dependencies', 'styling' ]
+		[ 'clean:build', 'lint', 'copy', 'concat:dependencies', 'styling' ]
```

Comment:
Is now a good time to drop linting too? Will always be passing on master branch, so just adds time to the deploy.